### PR TITLE
fix: various styling fixes for devtools and narrow

### PR DESCRIPTION
**Pull request by patrickhulce**
easier to support devtools and narrowness as class-based instead of media queries

cc @paulirish

### CODE REVIEW

File: lighthouse-core/report/v2/renderer/report-ui-features.js [hunk 1]
```diff
@@ -71,6 +73,25 @@ class ReportUIFeatures {
...
   }
 
+  _setupMediaQueryListeners() {
+    const mediaQuery = self.matchMedia('(max-width: 600px)');
+    mediaQuery.addListener(this.onMediaQueryChange);
+    this.onMediaQueryChange(mediaQuery);
+  }
+
+  /**
+   * Handler media query change events.
```
Comment (ebidel) [inline lighthouse-core/report/v2/renderer/report-ui-features.js]: Handler -> Handle

File: lighthouse-core/report/v2/renderer/report-ui-features.js [hunk 2]
```diff
@@ -71,6 +73,25 @@ class ReportUIFeatures {
     this._document.dispatchEvent(event);
   }
 
+  _setupMediaQueryListeners() {
+    const mediaQuery = self.matchMedia('(max-width: 600px)');
+    mediaQuery.addListener(this.onMediaQueryChange);
+    this.onMediaQueryChange(mediaQuery);
```
Comment (ebidel) [inline lighthouse-core/report/v2/renderer/report-ui-features.js]: are you calling this to kick it on init or is this for testing? If it's the former, can you add a comment?

File: lighthouse-core/report/v2/renderer/report-ui-features.js [hunk 3]
```diff
@@ -71,6 +73,25 @@ class ReportUIFeatures {
...
+  }
+
+  /**
+   * Handler media query change events.
+   * @param {!MediaQueryList} mql
+   */
+  onMediaQueryChange(mql) {
+    const root = this._dom.find('.lh-root', this._document);
+    if (mql.matches) {
```
Comment (ebidel) [inline lighthouse-core/report/v2/renderer/report-ui-features.js]: one liner: `root.classList.toggle('lh-narrow', mql.matches);`

**Comment by paulirish** [comment]
can you tell who here was an html5 advocate? 😁

File: lighthouse-core/report/v2/renderer/report-ui-features.js (hunk 1 above)
Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/report-ui-features.js]: done, fixed copy too :)

File: lighthouse-core/report/v2/renderer/report-ui-features.js (hunk 2 above)
Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/report-ui-features.js]: done

File: lighthouse-core/report/v2/renderer/report-ui-features.js (hunk 3 above)
Comment (patrickhulce) [inline lighthouse-core/report/v2/renderer/report-ui-features.js]: done

**Pull request merged**